json/src/spec/account.rs

```diff
@@ -18,7 +18,7 @@
 
 use std::collections::BTreeMap;
 
-use crate::{bytes::Bytes, spec::builtin::Builtin, uint::Uint};
+use crate::{bytes::Bytes, spec::builtin::BuiltinCombat, uint::Uint};
```

Comment:
Combat? Did you mean "compat"? I mean, it's a struggle for sure, I feel your pain! :)

Reply:
Yes, thanks